core/src/main/java/org/opensearch/sql/data/model/ExprDateValue.java

```diff
@@ -67,7 +69,7 @@ public LocalDateTime datetimeValue() {
 
   @Override
   public Instant timestampValue() {
-    return ZonedDateTime.of(date, timeValue(), ZoneId.systemDefault()).toInstant();
+    return ZonedDateTime.of(date, timeValue(), ZoneId.of("UTC")).toInstant();
```

Comment:
Last minor change - you need to revert this as well ... or ... update test to use `UTC` too.